x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/utils/validate.ts

```diff
@@ -95,9 +96,10 @@ export const validateResponseActionsPermissions = async (
 
   const endpointAuthz = await securitySolution.getEndpointAuthz();
 
-  const differences = findDifferenceInArrays<ResponseAction, RuleResponseAction>(
+  const differences = xorWith<ResponseAction | RuleResponseAction>(
```

Comment:
I like this better indeed. I was going to suggest `xorWith` earlier but didn't want to ask you to change too much. 🎉 Thanks for updating this. I think you should name the varaible `symmetricDifference` anyway. Maybe, also leave a comment above that says `finds elements that are not in either array`, so one doesn't have to look up the function to understand what this does.